Linux support

Hi,
compiling your example code gives:
```
error[E0432]: unresolved imports `clipboard_master::Master`, `clipboard_master::ClipboardHandler`, `clipboard_master::CallbackResult`
 --> src/main.rs:3:24
  |
3 | use clipboard_master::{Master, ClipboardHandler, CallbackResult};
  |                        ^^^^^^  ^^^^^^^^^^^^^^^^  ^^^^^^^^^^^^^^ no `CallbackResult` in the root
  |                        |       |
  |                        |       no `ClipboardHandler` in the root
  |                        no `Master` in the root
```

--- a/Cargo.toml
+++ b/Cargo.toml
@@ -23,7 +23,7 @@ features = [
 ]
 
 [target.'cfg(all(unix, not(any(target_os="macos", target_os="ios", target_os="android", target_os="emscripten"))))'.dependencies]
-x11-clipboard = "0.3"
+x11-clipboard = "0.5"
 
 [target.'cfg(target_os = "macos")'.dependencies]
 objc = "0.2"

--- a/src/master/x11.rs
+++ b/src/master/x11.rs
@@ -17,7 +17,7 @@ impl<H: ClipboardHandler> Master<H> {
         xcb::xfixes::query_version(&clipboard.getter.connection, 5, 0);
 
         loop {
-            let selection = clipboard.getter.atoms.utf8_string;
+            let selection = clipboard.getter.atoms.clipboard;
 
             let screen = match clipboard.getter.connection.get_setup().roots().nth(clipboard.getter.screen as usize) {
                 Some(screen) => screen,
@@ -37,22 +37,16 @@ impl<H: ClipboardHandler> Master<H> {
                                                 xcb::xfixes::SELECTION_EVENT_MASK_SELECTION_CLIENT_CLOSE |
                                                 xcb::xfixes::SELECTION_EVENT_MASK_SELECTION_WINDOW_DESTROY);
             clipboard.getter.connection.flush();
-            let first_event = xfixes.first_event();
 
             loop {
-                let event = match clipboard.getter.connection.wait_for_event() {
-                    Some(event) => event,
+                match clipboard.getter.connection.wait_for_event() {
+                    Some(_) => {
+                        break
+                    },
                     None => {
                         continue
                     }
                 };
-
-                let rsp_type = event.response_type();
-
-                match rsp_type & !0x80 {
-                    xcb::SELECTION_NOTIFY | xcb::PROPERTY_NOTIFY => break,
-                    _ => (),
-                }
             };
 
             match self.handler.on_clipboard_change() {

